.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: npm ci
 
       - name: Lint
-        run: ./_tests/css.sh
+        run: npx stylelint "_styles/**/*.css" -f github
```

Comment:
What is the reason for having the formatter specified here? Does it not work if you just allow the default setting?

Reply:
This is the console output format of the lint errors.

This one is machine readable by GitHub actions and allows GitHub to show errors inline against specific lines during the PR process.

We wouldn't want to make this the default as it's much better to have the nice pretty coloured human readable output for local development.